login.php

```diff
@@ -23,10 +23,11 @@
  *
  *  fUsername
  *  fPassword
+ *  token
  *  lang
  */
 
-define('POSTFIXADMIN_LOGOUT', 1);
+//define('POSTFIXADMIN_LOGOUT', 1);
```

Comment:
Feel free to adjust the logout code with your proposal ;-)

I just tested your change and always get "invalid token" when trying to login :-( Can you please check what is wrong?